sources/api/migration/migrations/v1.18.1/bootstrap-containers-config-file-v0-1-0/src/main.rs

```diff
@@ -0,0 +1,23 @@
+use migration_helpers::common_migrations::{
+    AddPrefixesMigration, ListReplacement, ReplaceListsMigration,
+};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
+        "configuration-files.bootstrap-containers-toml",
+    ]))?;
+    migrate(ReplaceListsMigration(vec![ListReplacement {
+        setting: "services.bootstrap-containers.configuration-files",
+        old_vals: &["host-ctr-toml"],
+        new_vals: &["host-ctr-toml", "bootstrap-containers-toml"],
+    }]))
+}
```

Comment:
Need to split this into two migrations to avoid a double `migrate` call (per https://github.com/bottlerocket-os/bottlerocket/pull/3715#discussion_r1466525851).